### DISCUSSION TITLE
Deployable MK19 Automatic Greanade Launcher

### DISCUSSION THREAD

**Discussion by hardguy334** [UncreatedStaff/UncreatedWarfare#203] [Ideas]
I think there should be another deployable like this. It's an infantry destroyer machine. Its basically a machine gun but with nade launcher heres the details: https://en.wikipedia.org/wiki/Mk_19_grenade_launcher
![mk19](https://user-images.githubusercontent.com/129318137/228616726-2f1ef7db-df95-482d-95a6-767a3c72d00b.jpg)

**Comment by hardguy334** [comment]
this is not even that bad if it is op it can be 10 ammo fob supply per /ammo (because it is shooting bombs) and 15 building supplies

**Comment by hardguy334** [comment]
when

**Comment by DanielWillett** [comment]
Haven't even been checking discussions ngl. If you want 420 to see this put it in issues.
